frontend/src/components/landing/PlanMatrix.module.scss

```
@@ -42,6 +42,22 @@
       display: none;
     }
   }
+
+  .table-wrapper {
+    padding: $layout-md 0;
+  }
+
+  .discount-notice-wrapper {
+    line-height: 100%;
+    text-align: center;
+
+    @include text-body-sm;
```

suggestion (nit): Put all `@include` calls at the top of the rule. I actually thought we had a Stylelint rule in place, but after looking around, that isn't the case. 🤷 
```suggestion
    @include text-body-sm;
    line-height: 100%;
    text-align: center;
```